Firestore/Example/FuzzTests/FuzzingTargets/FSTFuzzTestSerializer.mm

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+#import "FSTFuzzTestTargets.h"
+
+#include "Firestore/core/src/firebase/firestore/model/database_id.h"
+#include "Firestore/core/src/firebase/firestore/remote/serializer.h"
+
+using firebase::firestore::model::DatabaseId;
+using firebase::firestore::remote::Serializer;
+
+int firebase::firestore::fuzzing::FuzzTestDeserialization(const uint8_t *data, size_t size) {
```

Comment:
You can do this, but for consistency please change to the form used elsewhere in the project:
```cpp
namespace firebase {
namespace firestore {
namespace fuzzing {

int FuzzTestDeserialization(/* and so on...*/

}
}
}
```
(it may look more verbose in this particular case, but it actually saves a lot of repetition in a file that has more functions).

Reply:
Sounds good.